src/components/templates/PartyMap/components/Map/Map.tsx

```diff
@@ -234,36 +234,34 @@ export const Map: React.FC<PropsType> = ({
   }
 
   return (
-    <div className="party-map-content-container">
-      <div className="party-map-container">
-        <div className="party-map-content">
-          <img
-            width="100%"
-            className="party-map-background"
-            src={venue.mapBackgroundImageUrl}
-            alt=""
-          />
+    <div className="party-map-page-container">
+      <div className="party-map-content-container">
```

Comment:
Are there specific reasons why we need to keep this nesting? And if so, wht are they?

Reply:
I have removed one level of nesting so far. `party-map-page-container` has sidebar(chat) inside

Reply:
Ah, makes sense. Eventually we want to refactor the sidebar out of here, but I guess we'll need to keep that for now.